saga/seata-saga-engine/src/main/java/io/seata/saga/engine/pcext/handlers/ServiceTaskStateHandler.java

```diff
@@ -239,6 +239,13 @@ private Object compensateSubStateMachine(ProcessContext context, ServiceTaskStat
         return interceptors;
     }
 
+    @Override
+    public void addInterceptor(StateHandlerInterceptor interceptor) {
+        if (interceptors != null && !interceptors.contains(interceptor)) {
+            interceptors.add(interceptor);
```

Comment:
Whether need to sort the interceptor?

Reply:
Sorting has been done in EnhancedServiceLoader.findAllExtensionDefinition method